### PR TITLE
"fix" too much space before content in Wiki

### PR DESCRIPTION
**Pull request by NobodysNightmare**
## OpenProject work package

https://community.openproject.org/work_packages/19558
## Description

I went for the dirty route... 'nuff said...

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://nobodysnightmare-feature-wiki-space-fix.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://nobodysnightmare-feature-wiki-space-fix.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/views/layouts/_action_menu_base.html.erb [hunk 1]
```diff
@@ -28,6 +28,12 @@ See doc/COPYRIGHT.rdoc for more details.
 ++#%>
 
 <% action_menu_key = action_menu_key ? action_menu_key : :action_menu_main %>
+<%
+   # TODO FIXME OMG! this is the ugliest hack I ever performed
+   # We need to hide the clearfix in the wiki to avoid additional spacing in the wiki
+   # THIS HACK NEEDS TO BE REPLACED BY AN ENGINEERS SOLUTION!
+   @no_clearfix ||= false
```
Comment (0xF013) [inline app/views/layouts/_action_menu_base.html.erb:35]: @NobodysNightmare isn't this equivalent to `@no_clearfix = false` ?

**Comment by myabc** [comment]
@NobodysNightmare my plan for the hack was to add back a horizontal rule after the action menu, but your solution is valid (and possibly better).

 :warning: **This will further increase our technical debt**, in case that concerns anyone.

Will merge once Travis is through though.

File: app/views/layouts/_action_menu_base.html.erb (hunk 1 above)
Comment (0xF013) [inline app/views/layouts/_action_menu_base.html.erb:35]: nvm, too much tooth ache

**Comment by NobodysNightmare** [comment]
> in case that concerns anyone.

pfft... technical whatsoever ^^

File: app/views/layouts/_action_menu_base.html.erb (hunk 1 above)
Comment (floriank) [inline app/views/layouts/_action_menu_base.html.erb:35]: You could remove `@no_clearfix` and just check for 

```
@no_clearfix.present?
```

in the condition AFAIK.

**Comment by floriank** [comment]
Technical debt _is_ a concern for everyone here - and we will plan the real solution in for 4.2 (or even 4.1.1).

This PR is yet another stain on the code base.

@NobodysNightmare:
1. please create a note in the respective WorkPackage's EPIC
2. rebase this, once #2925 has been approved and merged to allow travis to run again
3. get a tattoo of this fix on a prominently visible part of your body

**Comment by myabc** [comment]
image

**Comment by 0xF013** [comment]
:+1: codewise

**Comment by myabc** [comment]
MySQL specs failing for non-PR related reasons.

**Comment by myabc** [comment]
This causes a regression in the old work packages view:
seeded_project_-_phase__815__sapiente_incidunt_aut_et_rerum_officia_dolor_molestias_-_openproject

**Comment by floriank** [comment]
What is the exact regression here? Too much spacing above the menu?

**Comment by NobodysNightmare** [comment]
For the record: After further examination we found out that this was not the cause.

In the offending places the `<div class="clearfix" />` is still present. I still don't know what causes the regression, though.

**Comment by myabc** [comment]
@NobodysNightmare regression is down to 5dea05c2a60f6d517b8d8d6130f8832af137abe7. We should revert this commit.

**Comment by NobodysNightmare** [comment]
@myabc Feels like I would need to google the meaning of each line there xD At least the last two...

However, I can't imagine that the fix adressed by that commit has something to do with our `clearfix`. Is that CSS class used to obsessively (i.e. naming conflict)? Or am I missing some obvious point there...

**Comment by myabc** [comment]
@NobodysNightmare I'm 99% sure that's the cause of the issue.

**Pull request merged**